web/src/main/java/org/fao/geonet/monitor/link/LinkCheckerUtils.java

```diff
@@ -0,0 +1,48 @@
+package org.fao.geonet.monitor.link;
+
+import org.jdom.Element;
+import org.jdom.JDOMException;
+import org.jdom.xpath.XPath;
+
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+public class LinkCheckerUtils {
+    public static String URL_XPATH = "gmd:linkage";
+    public static String NAME_XPATH = "gmd:name";
+
+    public static boolean checkHttpUrl(String url) {
+        try {
+            HttpURLConnection connection;
+            connection = (HttpURLConnection) (new URL(url)).openConnection();
+            connection.setConnectTimeout(LinkMonitorService.timeout * 1000);
+            connection.setReadTimeout(LinkMonitorService.timeout * 1000);
+            connection.setRequestMethod("GET");
+            connection.connect();
+
+            LinkMonitorService.getLogger().debug(String.format("%s -> %d", url, connection.getResponseCode()));
+            if (connection.getResponseCode() != 200)
+                LinkMonitorService.getLogger().debug(String.format("URL '%s' is unavailable", url, connection.getResponseCode()));
```

Comment:
I might see how to initialize the logger better, you can show me perhaps. Is it a new instance per class?